news-TS/src/components/app/app.ts

```diff
@@ -0,0 +1,24 @@
+import AppController from '../controller/controller';
+import { AppView } from '../view/appView';
+import { IAppView, IApiResponse, IAppController } from '../../types/index';
+
+class App {
+    controller: IAppController;
+    view: IAppView;
+    constructor() {
+        this.controller = new AppController();
+        this.view = new AppView();
+    }
+
+    start() {
+        const mainSources = document.querySelector('.sources') as HTMLDivElement;
```

Comment:
This thing is not fully correct, when you use `as` you are missing the part when there is no such div on the page. If there is no such div then the code below is potentially buggy and typescript is silent about it. So, ofter `as` is dangerous as `any`.
You can try to compare your code with:
```
const mainSources = document.querySelector<HTMLDivElement>('.sources');
```
You will see that TS does not allow you to set `addEventListener` because `mainSources` might be null, so you have to add additional if and you will be more safe then.